src/components/PolandMap.tsx

```diff
@@ -45,29 +45,47 @@ export const PolandMap = ({ cities, events = [] }: MapProps) => {
                         return (
                             <g key={index}>
                                 <Link href={city.href}>
-                                    <circle
-                                        cx={city.pointPosition.x}
-                                        cy={city.pointPosition.y}
-                                        r="3"
-                                        fill={
-                                            city.status === 'active' ? '#219eab' :
-                                                city.status === 'coming-soon' ? '#EAB308' :
-                                                    city.status === 'new' ? '#219eab' :
-                                                        '#9CA3AF'
-                                        }
-                                        className="cursor-pointer hover:opacity-80"
-                                    />
-                                    {eventStatus !== 'none' && (
+                                    <g className="city-marker group">
```

Comment:
It would be nice to have those markers as a components accepting city data and status in the future.